mep/people/templates/people/member_cardlist.html

```diff
@@ -0,0 +1,60 @@
+{% extends 'base.html' %}
+{% load render_bundle from webpack_loader %}
+{% load static mep_tags %}
+
+{% block page-subtitle %}{{ member.firstname_last }} Cards · Library Members · {% endblock %}
+
+{% block js %}
+{% render_bundle 'activities' 'js' %}
+{% endblock %}
+
+{% block header %}
+{% include 'snippets/header.html' with title=member.firstname_last style="detail" %}
+{% endblock %}
+
+{% block main-style %}tabbed white{% endblock %}
+
+{% block content %}
+{% include 'snippets/breadcrumbs.html' %}
+<nav class="tabs" aria-label="sections">
+    <li class="tab">
+        <a href="{% url 'people:member-detail' slug=member.slug %}">biography</a>
+    </li>
+    <li class="tab">
+        <a href="{% url 'people:membership-activities' slug=member.slug %}">activities</a>
+    </li>
+    <li class="tab" aria-selected="true">
+        <span>cards</span>
+    </li>
+</nav>
+
+<section class="member-cards">
+    <ol class="card-results">
+    {% for card in cards %}
+    <li class="card result">
+        {# <a href="#" aria-label=""> #}
+                {# card thumbnails and year if known #}
+                <picture>
+                  {% with card.image|iiif_image:"size:width=225" as 1xthumbnail %}
+                    <source srcset="{{ 1xthumbnail }}, {{ card.image|iiif_image:"size:width=450" }} 2x">
+                    <img src="{{ 1xthumbnail }}" alt="">
+                  {% endwith %}
+                </picture>
+                {% with card.footnote_set.event_date_range as dates %}
+                <div class="card-dates {% if not dates %}unknown{% endif %}">
+                    <dt>Card Years</dt>
+                    <dd>{% if dates.0 %}{{ dates.0.year }}
+                        {% if dates.0.year != dates.1.year %}–<span class="sr-only">to</span> {{ dates.1.year }}{% endif %}
+                        {% else %}Unknown{% endif %}
+                    </dd>
+                </div>
+                {% endwith %}
+        {# </a> #}
+    </li>
+    {% empty %}
+    <li>No cards available.</li>
```

Comment:
seems odd to render the absence of an element as an element? I would just render a text element before or after the empty `<ol>`, or just not render the `<ol>` at all.

Reply:
yeah, fair enough - I always find myself wanting to use the `for` / `empty` pattern, but it usually doesn't make sense